po/extra/fr.po

```diff
@@ -0,0 +1,150 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the extra package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: extra\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-12-24 21:21+0000\n"
+"PO-Revision-Date: 2022-06-20 17:15+0200\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"Language: fr\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+"X-Generator: Poedit 3.0.1\n"
+
+#: data/com.github.childishgiant.mixer.desktop.in:4
+#: data/com.github.childishgiant.mixer.appdata.xml.in:7
+msgid "Mixer"
+msgstr "Mixeur"
+
+#: data/com.github.childishgiant.mixer.desktop.in:5
+#: data/com.github.childishgiant.mixer.appdata.xml.in:8
+msgid "Change the volume of apps"
+msgstr "Modifier le volume des applications"
+
+#: data/com.github.childishgiant.mixer.desktop.in:6
+msgid "Volume mixer"
+msgstr "Mixeur de volume"
+
+#: data/com.github.childishgiant.mixer.desktop.in:8
+msgid "com.github.childishgiant.mixer"
+msgstr "com.github.childishgiant.mixer"
+
+#: data/com.github.childishgiant.mixer.desktop.in:11
+msgid "Sound;Audio"
+msgstr "Sound;Audio;son;mixeur;volume;"
+
+#: data/com.github.childishgiant.mixer.appdata.xml.in:10
+msgid "A no-frills volume mixer, with simplicity and usability at its core."
+msgstr ""
+"Un mélangeur de volume sans fioritures, avec la simplicité et la facilité "
+"d'utilisation au cœur."
+
+#: data/com.github.childishgiant.mixer.appdata.xml.in:12
+msgid "Simple volume controls including balance and output selection"
+msgstr ""
+"Des commandes de volume simples, y compris la sélection de la balance et "
+"de la sortie"
+
+#: data/com.github.childishgiant.mixer.appdata.xml.in:13
+msgid "Individually change each audio source's values"
+msgstr "Changez individuellement les valeurs de chaque source audio"
+
+#: data/com.github.childishgiant.mixer.appdata.xml.in:19
+msgid "Allie Law"
+msgstr "Allie Law"
+
+#: data/com.github.childishgiant.mixer.appdata.xml.in:32
+msgid "Nothing new!"
+msgstr "Rien de nouveau !"
```

Comment:
Is the space at the end needed?

Reply:
Yes, in French there is always a space between words and some punctuation signs like «, », !, ?, ; ...